### DISCUSSION TITLE
maFile

### DISCUSSION THREAD

**Discussion by Kajo576** [dyc3/steamguard-cli#231] [Q&A]
I inserted the mafile to the folder where is the manifest.json
But i see its not enough. I need edit the manifest file? how can i add my mafile to the entries?
![355963232_800244295052908_7052168975757654534_n](https://github.com/dyc3/steamguard-cli/assets/39519799/9fb42681-0f17-4c69-bb18-97eb66112d00)

**Comment by dyc3** [comment]
Could you post the output with `-v debug`? Preferably _without_ the file names and directories censored.
